### ISSUE TITLE
[QUESTION]  add schema registry url

### ISSUE THREAD

**Issue by hamroune** [aio-libs/aiokafka#616]
Hi everyone, I can't find option to set schema registry url, can some one help me to point the right way to add Avro Schema based on Schema Registry?
thanks

**Comment by gabriel-tincu** [comment]
I don't believe that any of the 2 clients (sync and async) support schema registry integration at the moment. That being said , you can peek at my failed attempt to create a schema registry serializer here https://github.com/aiven/karapace/blob/performance-improvements/karapace/serialization.py . 

It's failed because we ended up needing a different interface that the one currently provided by the python client. And the http calls made to the registry were async, so the serialize/deserialize methods ended up being async as well. 

That being said, the serializer can easily be adapted to fit said interface, provided one uses only a subject name strategy. Hope this clears things up somewhat

**Comment by r-owen** [comment]
[kafkit](https://kafkit.lsst.io) is one option for a schema registry interface
